docs/website/.npmrc

```diff
@@ -0,0 +1 @@
+engine-strict=true
```

Comment:
do we really need this? we don't fix npm version in package.json

Reply:
Node version range is already specified in `package,json` in the `engines` section.   

Reason for my change was that without the `engine-strict=true` it just produces a version warning, which for me as a new Node user was lost in a lot of other frightening output.  Even a recent edition of Ubuntu (22.04) only has Node 12.22, which is older than the package's required `>=16.14".  

With `engine-strict=true`  it simply refuses to install and succinctly tells the user that they need a later version of Node.  The website README helps them get the latest stable version.  

I have not considered how making it strict could affect broader CI/CD processes. ie, are there other parts of dlt that depend on Node.  

Background info:
- https://docs.npmjs.com/cli/v10/configuring-npm/package-json#engines 
- upgrading node also upgrades its bundled npm, so fixing node version indirectly fixes npm version too.

Reply:
let's try =)